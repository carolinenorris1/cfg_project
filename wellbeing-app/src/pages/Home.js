// import styles from "../globalStyles.module.css";

function Home() {
  return (
    <div>
      <h1>Home content</h1>
    </div>
  );
}
export default Home;
