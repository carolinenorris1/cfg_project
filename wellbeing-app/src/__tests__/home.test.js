import React from "react";
import renderer from "react-test-renderer";

import Home from "../pages/Home";

it("renders correctly when user runs the app", () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
