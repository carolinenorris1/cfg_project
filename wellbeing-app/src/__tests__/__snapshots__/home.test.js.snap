// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when user runs the app 1`] = `
<div>
  <img
    alt="Photograph of woman sitting on a bench outside, working on a laptop."
    className="hero-image"
    src="images/home-images/hero.jpg"
  />
  <div
    className="slogan"
  >
    <p>
      Time to UnHook
    </p>
  </div>
  <div
    className="main-page-txt-container"
  >
    <div
      className="main-page-left-container"
    >
      <p>
        We see you. 
        <i>
          Tip tapping
        </i>
         on that keyboard all day. Peering into the abyss. We both know that all that screen time isn't good for you: eye strain, neck pain, Carpal tunnel... What if we could take it all away?
      </p>
      <button
        onClick={[Function]}
      >
        Check it out ðŸ‘‰
      </button>
    </div>
    <div
      className="main-page-right-container"
    >
      <p
        className="mission"
      >
        Our Mission
      </p>
      <ul>
        <li>
          To remind you when to give yourself, and your eyes, a little break (because we know you forget)
        </li>
        <li>
          To stop working from home being so painful (literally) 
        </li>
        <li>
          To help you be the healthiest & most productive version of yourself
        </li>
        <li>
          To smile at strangers (because it's nice)
        </li>
      </ul>
      <button
        onClick={[Function]}
      >
        Let us help ðŸ‘‰
      </button>
    </div>
  </div>
</div>
`;
